server/views/pages/goal/update/review.njk

```diff
@@ -0,0 +1,131 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "review-update-goal" %}
+{% set pageTitle = applicationName + " - Home" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      },
+      {
+        text: "Manage learning and work progress",
+        href: "/"
+      },
+      {
+        text: "Manage " + prisonerSummary.firstName | capitalize + " " + prisonerSummary.lastName | capitalize + "'s learning plan",
+        href: "/plan/" + prisonerSummary.prisonNumber + "/view/overview"
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% include "../../../partials/prisonerBanner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">Check your updates for {{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s goals</h1>
+
+      <h2 class="govuk-heading-m govuk-visually-hidden">Goal details</h2>
+
+      {% if data.status == "COMPLETE" %}
```

Comment:
This is not quite right .... unfortunately we have 2 types of status we are dealing with here - Goal Status and Step Status .... and unfortunately they have different values! 😢 

The field you are dealing with in _this_ part of the template is the Goal Status - and it has these values:
```
'ACTIVE' | 'COMPLETED' | 'ARCHIVED'
```
And just to make it more confusing, the terminology they like to use for an Active goal is "In progress" ... not "Started" ! 
We will need to find out the complete set of corresponding screen labels/terminology (I dont know them off hand)

---

Steps have their own status enum, which is subtly different! It has these values:
```
'NOT_STARTED' | 'ACTIVE' | 'COMPLETE'
```
Where the corresponding screen labels/terminology are "Not started", "Started", "Completed"

---

Rather than putting this logic in the template as you have here for Goal Status; and further down for Step Status on line 90 - can you please write a couple of filters for these please? One for Goal Status, and one for Step Status.
Have a look at `formatDateFormValue` and it's unit test as an example.

Reply:
Think there is a little bit of inconsistency through the app (mainly Overview and this page) with the labeling/wording - will double check the designs and implement filters

Reply:
Confirmed the labels with Jo and created a filter for both Goal status and Step status 👍🏻 .